airtouch4pyapi/airtouch.py

```diff
@@ -34,16 +35,21 @@ class AirTouchStatus(Enum):
     CONNECTION_LOST = 3,
     ERROR = 4
 
+class AirTouchVersion(Enum):
+    AIRTOUCH4 = 4,
```

Comment:
having the comma here makes this into a tuple (classic python) so the .value function you use later does not work for Airtouch4 (comes out as (4,)) - I think if you just remove the comma from here (and the one above would be awesome as well) it should work

Reply:
I think fixed. Thanks!